packet/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,27 +1,27 @@
 module "bootkube" {
-  source = "github.com/kinvolk/terraform-render-bootkube?ref=30b2abcb1266ce9cedc92ddb3c6d8a4bbcbe7da0"
+  source = "git::https://github.com/kinvolk/terraform-render-bootkube.git?ref=mauricio/tf-0.12"
```

Comment:
Was done by the 0.12 upgrade script or changed by you? (i mean the `git::https` prefix)

Reply:
`git::https` is needed because ref is a branch.

Reply:
Oh, really? That sound weird... Is this documented somewhere? Or you saw an error that it had problems to disambiguate or something?

Reply:
I saw an error and then have to change. Actually I'm not sure what is the reason for this, given that it's a temporal change I didn't go deep in the details.

Reply:
Oh, right, this is temporal. Never mind! :)

Reply:
The following works fine for me:

```hcl
module "tf-test" {
  source = "github.com/johananl/tf-test?ref=branch"
}
```

It makes sense to me that all Git ref types (branch, tag, commit etc.) would work the same way.

Reply:
Huh. Looks like the problem is slashes in the ref field:

```
Error downloading modules: Error loading modules: error downloading 'https://github.com/johananl/tf-test.git?ref=my': /bin/git exited with 1: error: pathspec 'my' did not match any file(s) known to git
```

Reply:
This is a bug in the go-getter library which is used by Terraform to fetch modules - see https://github.com/hashicorp/go-getter/pull/228.